src/commands.rs

```diff
@@ -1239,25 +1242,43 @@ fn cmd_config(
     command: &CommandHelper,
     subcommand: &ConfigSubcommand,
 ) -> Result<(), CommandError> {
-    ui.request_pager();
```

Comment:
Split this change to separate commit? It's a bug fix for `jj config edit`.

It might be also good to split cmd_config_list/_edit() function since they are quite different in implementation.